utils/docker/Dockerfile.focal

```diff
@@ -13,7 +13,8 @@ RUN apt-get update && \
     # Create the pwuser
     adduser pwuser
 
-ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+ARG PW_TARGET_ARCH
```

Comment:
We should be able to use `uname -m` to detect target architecture.

Reply:
We can't set env vars dynamically with a Dockerfile so I saw no other way around it. :/ 
Or wdyt?

Reply:
Yeah, that's a bummer, I guess the best workaround at the moment is this:
```
RUN export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-$(dpkg --print-architecture)
```
Or even writing this env variable into .bashrc but I agree this is somewhat cumbersome.

Reply:
Yeah, that's a bummer, I guess the best workaround at the moment is this:
```
RUN export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-$(dpkg --print-architecture)
```
Or even writing this env variable into .bashrc but I agree this is somewhat cumbersome.

Reply:
> Yeah, that's a bummer, I guess the best workaround at the moment is this:
> 
> ```
> RUN export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-$(dpkg --print-architecture)
> ```
> 
> Or even writing this env variable into .bashrc but I agree this is somewhat cumbersome.

This won't work, since the export only sets it for the current active bash/sh session which does not re-used / envs get pulled out of it.

> Or even writing this env variable into .bashrc but I agree this is somewhat cumbersome.

Imagine a user does put "java -jar ..." as a Docker entrypoint, it would never invoke bash/sh so the custom env var does not get loaded / injected.

---

So I think having it with a custom arg might be the best solution for now.